docs/css/style-v19.css

```diff
@@ -719,3 +719,27 @@ body.no-sidebar .sidenav-trigger {
     text-decoration: none !important;
 }
 
+/* Auth and Data Provider icons */
+
+img {
```

Comment:
Not good: we have other images than just the ones you've added. Now all the images are VERY small! Only style under a subclass.